### PR TITLE
Add parameters to virt module.

### PR DESCRIPTION
**Pull request by RSpiertz**
This patch fixes issue #84

### CODE REVIEW

File: cloud/virt.py [hunk 1]
```diff
@@ -55,6 +55,26 @@
       - XML document used with the define command
     required: false
     default: null
+  xml_secure:
+    description:
+      - libvirt VIR_DOMAIN_XML_SECURE flag, only used for get_xml
+    required: false
+    default: 0
```
Comment (bcoca) [inline cloud/virt.py:62]: default should read False (I know 0 gets passed in later, but here you are defining a boolean to the user)

**Comment by bcoca** [comment]
could you rebase this?

**Comment by gregdek** [comment]
@RSpiertz A friendly reminder: this pull request has been marked as needing your action, and it hasn't been updated in the last month. 

If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we'll keep it open pending your changes.

If we receive no indication from you within a week, we will close this pull request.

**Comment by gregdek** [comment]
Closed due to contributor inactivity. Note to future contributors: if you would like to pick up this contribution, please do so by opening a new pull request rather than reopening this one. Thanks!

**Pull request closed without merging**